zap-desktop-getting-started.md

```diff
@@ -1,92 +1,67 @@
-# Zap Desktop - Getting started
+  # Zap Desktop - Getting started
 
-If you want to watch a video tutorial click [here]().
+  If you want to watch a video tutorial click [here]().
 
-This tutorial assumes you have already downloaded the new Zap Desktop v0.1.2 found [here](https://github.com/LN-Zap/zap-desktop/releases).
+  This tutorial assumes you have already downloaded the new Zap Desktop v0.3.0 found [here](https://github.com/LN-Zap/zap-desktop/releases).
 
-## Step 1: Connection
-Ok let's do this dance. Once you've started Zap you will see a screen titled "How do you want to connect to the Lightning Network?"
+  - Mac users will need to download ZapDesktop-darwin-v0.3.0-beta.dmg
+  - Windows users will need to download ZapDesktop-win32-v0.3.0-beta.exe
+  - Linux users will need to download one of the following:
+      - ZapDesktop-linux-amd64-v0.3.0-beta.deb
```

Comment:
There should be 4 files listed for linux - `.AppImage' and `.deb` each for 64 bit and 32 bit.

Reply:
Good catch!